cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/aws-orchestrator/AwsOrchestrator.service.tsx

```diff
@@ -78,7 +80,7 @@ export class AwsOrchestrator {
                 [AWS_NODE_PROFILE_NAMES.PRODUCTION_READY]: '',
             };
             Object.keys(nodeProfileList).forEach((nodeProfile) => {
-                nodeProfileList[nodeProfile] = AwsDefaults.setDefaultNodeType(nodeTypeList, nodeProfile);
+                nodeProfileList[nodeProfile] = findFirstOverlappingElement(nodeTypeList, getDefaultNodeTypes(nodeProfile));
```

Comment:
Last fallback if none found, select instance type from middle of list

Reply:
done